apps/dashboard/app/controllers/batch_connect/session_contexts_controller.rb

```diff
@@ -1,5 +1,6 @@
 class BatchConnect::SessionContextsController < ApplicationController
   include BatchConnectConcern
+  require "json"
```

Comment:
I don't think this should be necessary here, I believe json is already included.